client/components/forms/form-text-input/style.scss

```diff
@@ -1,7 +1,7 @@
 @import "@automattic/typography/styles/variables";
 @import "../../../assets/stylesheets/shared/extends-forms";
 
-.form-text-input:not(.form-text-input-core-styles) {
```

Comment:
This is the change that increased the specificity and made the rules override others in the codebase